rclc/CMakeLists.txt

```diff
@@ -1,56 +1,126 @@
 cmake_minimum_required(VERSION 3.5)
+set(CMAKE_C_CLANG_TIDY clang-tidy -checks=*)
 
 project(rclc)
 
+#################################################
+# compiler settings
+#################################################
+
+# Default to C11
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 11)
+endif()
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# for performance tracing
+# see https://gitlab.com/micro-ROS/ros_tracing/ros2_perf_analysis/tree/feature/simple-analysis/scripts
+#add_compile_options(-finstrument-functions
+#-fplugin=~/ros_ws/src/instrument-attribute-gcc-plugin/instrument_attribute.so
+#-fplugin-arg-instrument_attribute-debug)
+
+set(CMAKE_VERBOSE_MAKEFILE ON)
+
+#################################################
+# package dependencies
+#################################################
 find_package(ament_cmake_ros REQUIRED)
 find_package(rcl REQUIRED)
+find_package(rcutils REQUIRED)
+find_package(std_msgs REQUIRED)
```

Comment:
alphabetical sort

Reply:
Fixed in 54f0bbc54a33cc601764ab1d53fe3119d9e61c8b